content/concepts/transport.md

```diff
@@ -115,3 +115,190 @@ and some places in the codebase still use the "swarm" terminology.
+
+1. IP layer: the connection will run on the IP layer, which is responsible for 
+   packet routing. 
+2. Transport layer: TCP then runs on top of the IP layer to provide a reliable 
+   byte stream.
+3. Secure communication layer: Secure communication (i.e., TLS) runs on TCP to 
+   encrypt the bytes.
+   - Negotiation (SYN-ACK) of encryption parameters for TLS. Standard TLS over 
+     TCP requires 3 RTT.
+4. Application layer: HTTP runs on a secure transport connection to transfer 
+   information.
+   - Data starts to flow.
+
+Secure TCP-based connections offer a multi-layered approach for secure communication 
+over IP, whereas QUIC, by design, is an optimized transport consolidation of layers. 
+QUIC has to deal with TCP-like congestion control, loss recovery, and encryption. 
+Part of the application layer is also built directly into QUIC; when you
+run HTTP on top of QUIC; only a small shim layer exists that maps 
+[HTTP semantics](https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html) 
+onto QUIC streams.
+
+To establish a connection, QUIC assumes that the node sends the right address over
+the packet. QUIC saves one round-trip by doing this. If there is suspicion of an 
+attack, QUIC has a defense mechanism that can require a three-way handshake, but only 
+under particular conditions. A minimum packet size rule exists for the first packet to 
+ensure that small malicious packets like SYN packets cannot be sent and consume excessive 
+resources, as can be done with TCP.
+
+QUIC saves another round-trip in using TLS 1.3 by optimistically providing keyshares. 
+If you have established a connection before, the host can send you a session ticket 
+that can be used to establish a new secure connection instantly, without any round-trips.
+
+> Over the last several years, the IETF has been working on a new version of TLS, TLS 1.3.
+> Learn more about TLS 1.3 and how it is used in libp2p on the secure communication concept guide.
+
+<!-- to add link to secure comm guide, and later to the specific doc that covers TLS -->
+
+libp2p only supports bidirectional streams and uses TLS 1.3 by default (but can use other
+cryptography methods). The streams in libp2p map cleanly to QUIC streams.
+
+When a connection starts, peers will take their host key and create a self-signed CA 
+certificate. They then sign an intermediate chain using their self-signed CA and put it 
+as a certificate chain in the TLS handshake. View the full TLS specification
+[here](https://github.com/libp2p/specs/blob/master/tls/tls.md).
+
+At the end of the handshake, each peer knows the certificate of the other. The peer can 
+verify if the connection was established with the correct peer by looking up the first 
+CA certificate on the certificate chain, retreive the public key, and using it to calculate 
+the opposing peer ID. QUIC acts like a record layer with TLS 1.3 as the backend as TLS is 
```

Comment:
I think this paragraph can be removed, this is very specific to libp2p-tls and doesn’t matter unless you actually want to implement libp2p-tls (in which case you should use the spec).

Reply:
Also removed